### DISCUSSION TITLE
Negative consumption with solarpanels

### DISCUSSION THREAD

**Discussion by felixn-unity** [UtilitechAS/amsreader-firmware#610] [General]
Hi

Can it also measure if our solarpanels are producing power to the grid? - meaning negative consumption

Thanks
Felix

**Comment by ArnieO** [comment]
Yes, it measures and displays negative consumption (delivery to grid), both graphically and accumulated as separate value. 
Smart power meters have two separate sets of values for most electrical parameters: Import and Export. 
There are some variations between smart meters on how many parameters are reported, this firmware reads and reports all values that your power meter provides.

**Comment by felixn-unity** [comment]
Great and thanks a lot :) - I am planning to trigger charging of my EV/Tesla when solar are producing too much

Felix
